AST-1620 - WP blocks styles changes on preview of Tablet and mobile on Firefox when Astra theme is active. (#4262)
brainstormforce/astra: changelog.txt

- Fix: Search element not displaying in Mobile Offcanvas.
- Fix: Elementor Pro - WooCommerce Checkout Form's shipping details not look appropriate in editor mode as it looks on the frontend.
- Fix: Gutenberg editor - Layout settings is not visble for Group block.

> - Fix: Fixed block editor style issue with the responsive preview option on the post or page.

Reviewer: ```suggestion
- Fix: Theme's responsive styles not loading with the responsive preview option in the block editor.
```

Author: Resolved!
